types.go

```diff
@@ -185,10 +186,13 @@ func (l *Limits) Validate(vr *ValidationResults) {
 	}
 
 	if l.Src != "" {
-		ip := net.ParseIP(l.Src)
+		elements := strings.Split(l.Src, ",")
```

Comment:
Do we know if parser deals with spaces or should we trim the elements array here?

Reply:
didn't but added that